includes/Story_Renderer/HTML.php

```diff
@@ -215,6 +216,23 @@ protected function replace_html_head( $content ) {
 		return $content;
 	}
 
+	/**
+	 * Force home urls to http / https based on context.
+	 *
+	 * @param string $content String to replace.
+	 *
+	 * @return string
+	 */
+	protected function replace_url_scheme( $content ) {
+		$home           = get_home_url();
+		$current_scheme = is_ssl() ? 'https' : 'http';
+		$replace_scheme = is_ssl() ? 'http' : 'https';
+		$search         = set_url_scheme( $home, $replace_scheme );
+		$replace        = set_url_scheme( $home, $current_scheme );
```

Comment:
```suggestion
		$current_scheme = is_ssl() ? 'https' : 'http';
		$replace_scheme = is_ssl() ? 'http' : 'https';
		$search         = home_url( '', $replace_scheme );
		$replace        = home_url( '', $current_scheme );
```

Reply:
On that note: confusing that `$replace_scheme` is not related to / used by `$replace`

Reply:
Tweaked.